docs/enhancement-proposals/off-ep-0007.md

```diff
@@ -0,0 +1,116 @@
+```
+U_{correction} = [N (N-1)]^{-1} \sum_{i < j}^N \int_0^{\infty} dr [1-S(r)] [4 \pi r^2] U_{vdw}(r)
+```
+Here, `U_{correction}` is the long-range vdW correction, `N` is the number of vdW sites, `S(r)` is the switching function or cutoff function that assumes the value of `S(r) = 0` for `r \ge r_{cut}` and `S(r) = 1` for `r ~ 0`.
+
+In [OpenMM](http://docs.openmm.org/latest/api-python/generated/openmm.openmm.NonbondedForce.html#openmm.openmm.NonbondedForce.setUseDispersionCorrection),
+this is controlled via a method that accepts a boolean toggle. Similar options are implemented
+in [GROMACS](https://manual.gromacs.org/current/user-guide/mdp-options.html#mdp-DispCorr), AMBER
+(`nodisper`), [LAMMPS](https://docs.lammps.org/pair_modify.html), and likely other engines. However
+unlikely, if in the future other types of corrections than the current isotropic one are developed,
+this specification can be extended because the setting is a string value, not a boolean.
+
+Many of these engines also provided the capability to use LJPME.
+As with electrostatics, we do not specify the exact parameters used for PME (real-space cutoff, grid spacing, error tolerance);
+the SMIRNOFF spec instead specifies the Ewald sum is the true desired potential and the simulation engine must make a faithful
+approximation of this true potential, but can adjust settings or use other algorithms to achieve improved performance without introducing significant error.
+
+## Backward compatibility
+
+This change proposes adding an attribute to a section, so in a strict sense it is not
+backwards-compatible. (A parser that only knows about version 0.3 might not understand a serialized
+representation of a version 0.4 section because of this new attribute, and at very least the
+information would be lost in a down-conversion.) However, the recommended
+default value matches what is currently ubiquitous in SMIRNOFF implementations, so an up-converter
+from section 0.3 to 0.4 should be straightforward to write and safe to use. It is recommended that
+any 0.3 version section is up-converted to a 0.4 section with the only change being an added
+`long_range_treatment` attribute with its default value of `"isotropic"`.
+
+## Detailed description
+
+A `long_range_treatment` attribute is added to the `vdW` section, which is bumped to version 0.4.
+
+Supported values are
+  * `"none"`: No long-range vdW correction is applied.
+  * `"isotropic"`: An isotropic vdW correction, described below, is applied.
+  * `"LJPME"`: An Ewald sum is used (commonly referred to as LJPME) to treat the vdW potential in a periodic manner [1](https://doi.org/10.1063/1.464397) [2](http://dx.doi.org/10.1063/1.465608) [3](https://doi.org/10.1021/acs.jctc.5b00726). Note that this is only compatible with certain forms of the potential that involve sums of inverse even powers of `r`.
```

Comment:
A fundamental issue here is that isotropic and PME give statistically different results, especially for non-homogeneous systems.  This may need to be explicitly called out.